tests/test_taar_lite_guidguid.py

```diff
@@ -47,12 +61,12 @@
     ]
 
 
-@mock.patch('mozetl.taar.taar_lite_guidguid.load_training_from_telemetry',
+@mock.patch('mozetl.taar.taar_lite_guidguid.extract_telemetry',
             return_value=MOCK_TELEMETRY_SAMPLE)
 @mock_s3
-def test_load_training_from_telemetry(spark):
+def test_extract_telemetry(spark):
     # Sanity check that mocking is happening correctly.
-    assert taar_lite_guidguid.load_training_from_telemetry(spark) == MOCK_TELEMETRY_SAMPLE
+    assert taar_lite_guidguid.extract_telemetry(spark) == MOCK_TELEMETRY_SAMPLE
 
 # Exercise the only part of the ETL job happening outside of spark.
 def test_addon_keying():
```

Comment:
You should test the output of the transformation method, perhaps using the `df_equals` test-fixture found in `conftest.py`. Example usage in this repo:

https://github.com/mozilla/python_mozetl/blob/054b34e93f687d344c1b3e9ab613681166858ef1/tests/engagement/test_daily_retention.py#L69-L85

Testing the transformation should subsume this unit test. I've been starting to focus more on testing correctness of the API boundary (`transform`), but sanity checks for implementation details like this one help too.